package.json

```diff
@@ -19,10 +19,11 @@
     "test": "builder run npm:test"
   },
   "dependencies": {
-    "builder": "^2.0.1",
-    "builder-react-component": "0.0.4"
+    "builder": "2.1.0",
```

Comment:
Sort of? (Because of https://github.com/FormidableLabs/builder#versions-v1-v2-v3)

But, you know, I think I'll just go free for all (well `^` in the future...)